delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storages/serialization/KryoSerializerInit.scala

```diff
@@ -0,0 +1,33 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.storages.serialization
+
+import ch.epfl.bluebrain.nexus.delta.plugins.storages.serialization.KryoSerializerInit.PathSerializer
+import com.esotericsoftware.kryo.io.{Input, Output}
+import com.esotericsoftware.kryo.{Kryo, Serializer}
+import io.altoo.akka.serialization.kryo.DefaultKryoInitializer
+import io.altoo.akka.serialization.kryo.serializer.scala.ScalaKryo
+
+import java.nio.file.Path
+
+//TODO: partially ported from service module, we might want to avoid this duplication
+class KryoSerializerInit extends DefaultKryoInitializer {
```

Comment:
I think this will collide with the service one; the serializer has a single initialisation class.

Reply:
Yes, and it is not a problem. Maybe I can move it to the tests, since this is just used in order to pass the tests (the path kryo serialization is already covered in the service)

Reply:
it would be better to move to the tests then, yes